content/articles/how-to-create-horizontal-scrolling-menu-using-jquery-and-php/index.md

```diff
@@ -0,0 +1,216 @@
+
+Following the establishment of a connection, we create a query to retrieve menu information from the database table ("menu_items") and save it in the result variable, after which the data is printed as an array.
+``` php
+<?php
+$servernme= "localhost";
+$usernme = "root";
+$pword = "";
+$dbname ="menu2";
+try {
+  $conn = new PDO("mysql:host=$servernme;dbname=menu2", $usernme, $pword);
+  //   error mode exception
+} catch(PDOException $e) {
+  echo "no established connection: ";
+}
+// Create connection
+$conn = new mysqli($servernme, $usernme, $pword,$dbname);
+// Check connection
+if ($conn->connect_error) {
+  die("no established connection: ". $conn->connect_error);
+}
+$sql = "SELECT table_data FROM menu_items";
+$result = $conn->query($sql);
+if ($result->num_rows > 0) {
+  // output data of each row
+  while($row = $result->fetch_assoc()) {
+  }
+} else {
+  echo "no results found";
+}
+```
+The php code included in our html code fetches the information and display it as an array of list items in the web page header region.
+``` php
+<?php foreach($result as $res){ ?>
+                        <a class="mnu-items" href="#"><?php echo $res['table_data']; ?></a>
+```
+
+### Creating database
+To create a horizontal menu list, we'll need to create a database and store menu items, which will then be accessed and presented on our page. We'll create a simple SQL database called ' menu2 ' with one table called ' menu_items ' and two columns: 'Items_id' (our primary key) and 'items_name' (where we'll store dummy menu items). We can now embed information into our data set after it has been made. When adding new data to the database, the items id column is set to auto-increment.
+
+When practicing, you can also create arbitrary names for your database, table, and columns. For example, take a look at the SQL code below.
+  ``` sql
+   SET SQL_MODE = "NO_AUTO_VALUE_ON_ZERO";
+START TRANSACTION;
+SET time_zone = "+00:00";
+-- Database: `menu2`
+-- Table structure for table `menu_items`
+CREATE Database`menu2`;
+USE menu2;
+CREATE TABLE `menu_items` (
+  `item_id` int(11) NOT NULL,
+  `table_data` varchar(255) NOT NULL
+) ENGINE=InnoDB DEFAULT CHARSET=latin1;
+-- keying data for table `menu_items`
+INSERT INTO `menu_items` (`item_id`, `table_data`) VALUES
+(0, 'home'),
+(1, 'about'),
+(2, 'contact'),
+(3, 'Info'),
+(4, 'services '),
+(5, 'Help'),
+(6, 'countries'),
+(7, 'Cities'),
+(8, 'jobs'),
+(9, 'calendar '),
+(10, 'feed'),
+(11, 'dashboard '),
+(12, 'login'),
+(13, 'clients'),
+(15, 'support '),
+(16, 'donate ');
+-- Indexes for table `menu_items`
+ALTER TABLE `menu_items`
+  ADD PRIMARY KEY (`item_id`);
+-- AUTO_INCREMENT for table `menu_items`
+ALTER TABLE `menu_items`
+  MODIFY `item_id` int(11) NOT NULL AUTO_INCREMENT, AUTO_INCREMENT=25;
+COMMIT;
+  ```
+
+### File-structure
+Ensure you save your files in the htdocs folder in the XAMP installation directory as shown.![Demo-picture](./img/filestructure.png)

```

Comment:
Is this uploaded? but wait, you should discuss project structure at the beginning of the project setup?

Reply:
Rectified